pages.id/common/ld.md

```diff
@@ -0,0 +1,16 @@
+# ld
+
+> Menghubungkan file-file obyek berbarengan.
+> Informasi lebih lanjut: <https://sourceware.org/binutils/docs-2.38/ld.html>.
+
+- Menghubungkan file obyek tertentu yang tidak memiliki kebergantungan ke suatu perintah:
```

Comment:
Shouldn't the last work be executable instead of command?

Reply:
Agree. What's the proper term for executable (should it be translated)?

Reply:
I am not a native speaker. Also, I too couldn't find a word for it online. Maybe similar to file we can leave executable as it is. What do you think?